libraries/mbed/targets/hal/TARGET_WIZNET/TARGET_W7500x/rtc_api.c

```diff
@@ -0,0 +1,78 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2006-2013 ARM Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "rtc_api.h"
+
+time_t wiz_rtc_time;
+char rtc_enabled = 0;
+
+#ifdef __cplusplus
+extern "C"{
+#endif
+void PWM3_Handler(void)
```

Comment:
I added NVIC_SetVector function.
I'm sorry I should have checked the code but I have not.
I correct and replace tab/spaces.